### DISCUSSION TITLE
Project structure src/the_way

### DISCUSSION THREAD

**Discussion by jgarte** [out-of-cheese-error/the-way#147] [Q&A]
Hi,

What was the reason for creating this directory and structuring the project with it?

https://github.com/out-of-cheese-error/the-way/tree/master/src/the_way

Just trying to understand this simple thing. Any insight appreciated.

**Comment by Ninjani** [comment]
Hi,

This was to separate more generic functions from those specific to `the-way`. For example, `gist.rs` has a generic gist API wrapper while `the-way/gist.rs` has gist functions specific to `the-way`. Same for utils and language.

Hope this helps, open to suggestions for refactoring
